File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/PersonRoomAddressPerson.java
package com.kinvey.androidTest.model;

import com.google.api.client.json.GenericJson;
import com.google.api.client.util.Key;

/**
 * Created by yuliya on 11/28/17.
 */

public class PersonRoomAddressPerson extends GenericJson {

    @Key
    private String name;

    @Key
    private Room room;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Room getRoom() {
        return room;
    }

    public void setRoom(Room room) {
        this.room = room;
    }
}

File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/PersonRoomAddressPerson.kt
package com.kinvey.androidTest.model

import com.google.api.client.json.GenericJson
import com.google.api.client.util.Key

/**
 * Created by yuliya on 11/28/17.
 */

data class PersonRoomAddressPerson(
    @Key
    var name: String? = null,
    @Key
    var room: Room? = null
) : GenericJson()